### DISCUSSION TITLE
The hashpipe (`#|`) shortcut

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [quarto-dev/quarto-cli#528] [Q&A] [answered]
Hi all,

Thanks again for working on this project. I was wondering if there is already a shortcut for the hashpipe (`#|`) operator (whether in Rstudio or in VScode). If not then thought it could be a nice goodie.

Cheers.

**Comment by cscheid** [comment, marked as answer]
We have some smarts built in to RStudio: if you type the _first_ hash-pipe and then hit enter, the second one is automatically filled out. But in general, I don't think we have a single shortcut key to get it started.

Still, this is an excellent idea. Another good reason to do it is that we should be smart enough to use the right comment syntax in different language cells. 

For example, in `ojs` cells, the "hash-pipe" is really a "slash-pipe": `//|`. The general syntax is the one-line comment for the particular language, followed by a pipe and a space. We should totally be supporting this in a single shortcut. I'll create a github issue to track. Thank you!

**Reply by cscheid** [reply]
See https://github.com/quarto-dev/quarto-cli/issues/533

**Reply by camilogarciabotero** [reply]
Hi @cscheid 

Wow, that was a quick response! Thanks for considering it. I'll keep an eye on it.
